index.html

```diff
@@ -8,7 +8,7 @@
   </head>
   <body>
     <div id="app">
-      <canvas id="canvas" style="height: 600px; width: 800px; border: 1px black;"></canvas>
+      <canvas id="canvas" style="border: 1px black;"></canvas>
```

Comment:
So is this just maximising the canvas to the size of the browser window, rather than hard coding to 800x600?

Reply:
its being manually set to 960 x 540 in the JS file.

leaving in the hard coding here would be redundant